src/Illuminate/Support/Arr.php

```diff
@@ -166,11 +166,11 @@ public static function first($array, callable $callback, $default = null)
      * Return the last element in an array passing a given truth test.
      *
      * @param  array  $array
-     * @param  callable  $callback
+     * @param  callable|null  $callback
      * @param  mixed  $default
      * @return mixed
      */
-    public static function last($array, callable $callback, $default = null)
+    public static function last($array, callable $callback = null, $default = null)
     {
         return static::first(array_reverse($array), $callback, $default);
```

Comment:
Per JoostK comment, use this at the beginning to greatly speed up the "no callback" mode:
```
if (is_null($callback)) {
    return !empty($array) ? end($array) : value($default);
}
```